kpops/components/base_components/kubernetes_app.py

```diff
@@ -66,6 +68,12 @@ class KubernetesApp(PipelineComponent):
         description=describe_object(__doc__),
         exclude=True,
     )
+    validate_name: bool = Field(
+        default=True,
+        description=describe_attr("check_name", __doc__),
```

Comment:
outdated name? shouldn't it be the same name as the attribute itself?

Reply:
Absolutely shoud